dev-env/bin/dade-check-main-only-files

```diff
@@ -6,11 +6,11 @@
 # on a PR branch which touches those files.
 #
 
-DADE_SKIP_MAIN_ONLY_CHECK="${DADE_SKIP_MAIN_ONLY_CHECK:-$(git log -1 --format=%b | tr -d ' ' | grep '^run-all-tests:true$')}"
+DADE_SKIP_MAIN_ONLY_CHECK="${DADE_SKIP_MAIN_ONLY_CHECK:-$(git log -1 --format=%b | tr -d ' ' | grep '^run-all-tests: *true$')}"
```

Comment:
huh shouldn't `tr -d ' '` have removed all the spaces?

Reply:
I would guess that this fix doesn't work then :/

Reply:
Ah, sorry: I don't use precommit hooks, I assumed you had tested it when you approved the merge to main.

Reply:
Ah oops, I did not, the fix seemed so simple i assumed it would work :/

Reply:
Lets disable automerge for now to debug this

Reply:
I've done a bit of local testing and it seems to work as expected for me, both with and without this change (i.e. confirming what @akrmn said).